package/MDAnalysis/auxiliary/base.py

```diff
@@ -659,7 +767,7 @@ def calc_representative(self):
         ndarray
             Array of auxiliary value(s) 'representative' for the timestep.
         """
-        if self.cutoff == -1:
```

Comment:
this isn't dependent on the time_step (I'm reading through things quickly so maybe I'm missing something?) - the `-1` case needs to be supported for 2.4.0 if that's the case. In which case you need to also raise a DeprecationWarning about support for `-1` and also add a `.. deprecated` docstring entry (and a deprecated entry in the changelog).

Reply:
That's supported now because on init, the cutoff kwarg gets turned into `None` and `self.cutoff` is guaranteed to be `None` or ≥ 0.

Reply:
Ah yes sorry my bad, I completely missed the line 309 change.